cmd/chisel/cmd_info.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/jessevdk/go-flags"
+	"gopkg.in/yaml.v3"
+
+	"github.com/canonical/chisel/internal/setup"
+)
+
+var shortInfoHelp = "Show information about package slices"
+var longInfoHelp = `
+The info command shows detailed information about package slices.
+
+It accepts a white-space separated list of strings. The list can be
+composed of package names, slice names, or a combination of both. The
+default output format is YAML. When multiple package or slice names
+are provided, the output is a list of YAML documents, separated by
+three dashes (“---”).
```

Comment:
Are we sure we want non-ASCII / curly quotes in this context?
```suggestion
three dashes ("---").
```

Reply:
Nice catch! Must have been copied from the Google doc. Fixed in https://github.com/canonical/chisel/pull/101/commits/b886f473b8b48064e4138ed6022bf165721e43be.